backend/BackendMIIT/src/main/java/com/example/BackendMIIT/configuration/AppConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.example.BackendMIIT.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class AppConfig {
+
+    @Bean
+    public WebClient webClient() {
+        return WebClient.builder().baseUrl("https://localhost:8080").build();
```

Comment:
Base url should be taken out to the `application.properties`, because this url may change and the file (`application.properties`) exists for storing such variables

Reply:
Done